docker/data/logstash/env2yaml/env2yaml.go

```diff
@@ -93,6 +93,7 @@ func normalizeSetting(setting string) (string, error) {
 		"http.port",        // DEPRECATED: prefer `api.http.port`
 		"log.level",
 		"log.format",
+		"log.format.json_strict",
```

Comment:
I wonder if "strict" is the right word here - we're not doing anything other than splitting duplicate "message" fields, and multiple "message" fields is valid json. 

Maybe `log.format.json.fix_duplicate_message_fields`?

Reply:
Fixed by 923756094db5c8c89bbfe2115567cf98606b94dd